tests/Http/Controllers/Api/Projects/GetUsersWithAnnotationsProjectsTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Biigle\Tests\Modules\Largo\Http\Controllers\Api\Projects;
+
+use ApiTestCase;
+use Biigle\Tests\ImageAnnotationLabelTest;
+use Biigle\Tests\ImageAnnotationTest;
+use Biigle\Tests\ImageTest;
+use Biigle\Tests\VideoAnnotationLabelTest;
+use Biigle\Tests\VideoAnnotationTest;
+use Biigle\Tests\VideoTest;
+use Biigle\Tests\ProjectTest;
+use Biigle\Tests\VolumeTest;
+use Biigle\Tests\UserTest;
+use Biigle\MediaType;
+use Biigle\Role;
+
+class GetUsersWithAnnotationsProjectsTest extends ApiTestCase
```

Comment:
The test class names usually match the original classes.
```suggestion
class GetUsersWithAnnotationsTest extends ApiTestCase
```